src/generics/utils.js

```diff
@@ -477,12 +492,23 @@ const convertToInteger = (value) => {
 	return isNaN(value) ? false : parseInt(value, 10)
 }
 
+const compareLength = (value1, value2) => {
+	const returnValue = value1 === value2 ? 0 : value1 > value2 ? -1 : 1
+	return returnValue
+}
+
 const isLabelValuePair = (item) => {
 	return Array.isArray(item)
 		? item.every((subItem) => subItem && typeof subItem === 'object' && 'label' in subItem && 'value' in subItem)
 		: typeof item === 'object' && 'label' in item && 'value' in item
 }
 
+const validateTitle = (title) => {
```

Comment:
@priyanka-TL Let me know if this is still needed

Reply:
yes required for project creation